app/models/post.rb

```diff
@@ -1,2 +1,17 @@
 class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments, foreign_key: 'posts_id'
+  has_many :likes, foreign_key: 'posts_id'
+
+  def most_recent_comments
+    comments.limit(5).order('created_at desc')
+  end
+
+  after_save :update_counter
+
+  private
+
```

Comment:
Nice Job for implementing the `after_save`  to call the `update_counter` method. 👍